extended-app-support.html.md.erb

```diff
@@ -23,14 +23,19 @@ To enable cflinuxfs3 support in <%= vars.app_runtime_abbr %> 10.2.1, operators m
     * **Errands:** Configure Post-Deploy and Pre-Delete errands related to the buildpacks and stack.
     * **Resource Config:** Select the VM type to use for executing the errand.
 
-1. Click **Apply Changes** to roll out the Extended App Support tile.
+1. **Apply Changes** to roll out the Extended App Support tile.
 
-1. Upgrade the foundation to 10.2.1 patch version or higher.
+1. Once completed, navigate to the <%= vars.app_runtime_abbr %> tile, click on **Cloud Controller** section in the sidebar. Change **Available Stacks** to `cflinuxfs4 only`. [cflinuxfs4-only](./images/cflinuxfs3_upgrade.png)
+
+1.  *Apply Changes** to migrate to cflinuxfs3 provided by the Extended App Support tile.
```

Comment:
"Apply Changes" is a button.
UI elements like buttons are bolded, so you need 2 * on either side of the word.